frb_rust/src/dart_api_dl/include/dart_api.h

```diff
@@ -0,0 +1,3953 @@
+/*
```

Comment:
Indeed I guess we are only using one or two dart api files? So we can remove the others to keep it more maintainable

Reply:
due to dependencies we use all files except:
- dart_embedder_api.h
- dart_tools_api.h
- dart_io_api.h
- analyze_snapshot_api.h (dart api 2.1)

it's also easier to copy the entire dart-sdk/includes folder than to keep track of all the dependencies

Reply:
(see above, where I guess we do not need that .c thus no need for a lot of .h)